custom_components/tuya_local/devices/jiahong_et72w_thermostat.yaml

```diff
@@ -132,16 +127,17 @@ entities:
             value: C
           - dps_val: true
             value: F
+      - id: 112
+        type: integer
+        name: floor_temp_calibration
   - entity: sensor
     name: Runtime
+    icon: "mdi:update"
```

Comment:
Adding `class: duration` is probably also a good idea here.

Reply:
Done